angular.json

```diff
@@ -349,43 +349,6 @@
         }
```

Comment:
Can you add some useful information to the PR description?
When merging to develop, this should automatically be copied by GH, and used in the release notes

Reply:
Done